Gopkg.toml

```diff
@@ -41,6 +45,11 @@
   branch = "master"
   name = "github.com/spf13/cobra"
 
+[[constraint]]
+  branch = "master"
+  name = "github.com/cenkalti/backoff"
+
+
```

Comment:
I would like to sort this alphabetically. There is also an additional empty line.

Reply:
Sure done.